src/components/settings/UISettings.svelte

```diff
@@ -40,8 +41,14 @@
     <EnumOption
       name="Video side:"
       options={Object.keys(VideoSide)}
-      store={videoSide}
+      store={videoSideSetting}
     />
+    <div style="margin-bottom: 10px;">
+      <CheckOption
+        name="Override layout settings when entering vertical mode"
+        store={autoVertical}
+      />
+    </div>
```

Comment:
I'm wondering if there's a better way to word these, cause it might be confusing for the video side setting to show left/right even when it's being overwritten during vertical mode. 

I guess for starters "Override layout settings" can be changed to "Override video side" since the former suggests multiple settings will be overridden.

Reply:
Maybe it could be named `Enter vertical mode when window is thin`?